client/signup/steps/domains/index.jsx

```diff
@@ -390,12 +403,9 @@ export class RenderDomainsStep extends Component {
 	};
 
 	submitWithDomain = ( { googleAppsCartItem, shouldHideFreePlan = false, signupDomainOrigin } ) => {
-		const { step, flowName } = this.props;
+		const { step } = this.props;
 		const { suggestion } = step;
 
-		if ( shouldUseMultipleDomainsInCart( flowName ) && suggestion ) {
-			return this.handleDomainToDomainCart();
-		}
```

Comment:
This is to improve the readability of the code, [added in this commit](https://github.com/Automattic/wp-calypso/pull/84831/commits/82f8490c6fd1ddf691c1703d11fd3c91e790affc). It doesn't affect the way things works.